pkg/api/conversion.go

```diff
@@ -248,3 +252,16 @@ func Convert_unversioned_LabelSelector_to_map(in *metav1.LabelSelector, out *map
 	}
 	return err
 }
+
+func Convert_Slice_string_To_Slice_int32(in *[]string, out *[]int32, s conversion.Scope) error {
```

Comment:
Yes, individual PRs where custom conversions are added should at least provide a godoc explaining why the custom conversion was added. I would love to see a doc where we document all custom conversions but that's out-of-scope for this PR.